modules/src/ics02_client/events.rs

```diff
@@ -214,3 +218,19 @@ impl From<ClientMisbehavior> for IbcEvent {
         IbcEvent::ClientMisbehavior(v)
     }
 }
+
+/// Signals a recent upgrade of an on-chain client (IBC Client).
+#[derive(Debug, Deserialize, Serialize, Clone)]
+pub struct UpgradeClient(Attributes);
+
+impl UpgradeClient {
+    pub fn set_height(&mut self, height: Height) {
```

Comment:
Are there no other attributes we might want to expose?

Reply:
I didn't notice anything else being necessary here at the moment.